src/shared/translations/nl.json

```diff
@@ -189,7 +197,7 @@
   "authentication/views/registration-flow/r-3-stamboek___zo-gaan-wij-na-of-jij-een-actieve-lesgever-bent-aan-een-vlaamse-erkende-onderwijsinstelling": "Zo gaan wij na of jij een actieve lesgever bent aan een Vlaamse erkende onderwijsinstelling.",
   "authentication/views/registration-flow/r-4-manual-registration___aanvraagformulier": "Aanvraagformulier",
   "authentication/views/registration-flow/r-4-manual-registration___achternaam": "Achternaam",
-  "authentication/views/registration-flow/r-4-manual-registration___bevestiging": "Bedankt voor je aanvraag. Onze helpdesk bekijkt deze binnen de vijf werkdagen. Heb je\n ondertussen nog vragen of toevoegingen met betrekking tot je aanvraag? Formuleer deze dan in\n een reply op automatische bevestigingsmail die je krijgt van onze helpdesk.",
+  "authentication/views/registration-flow/r-4-manual-registration___bedankt-voor-je-aanvraag-onze-helpdesk-bekijkt-deze-binnen-de-vijf-werkdagen-heb-je-ondertussen-nog-vragen-of-toevoegingen-met-betrekking-tot-je-aanvraag-formuleer-deze-dan-in-een-reply-op-automatische-bevestigingsmail-die-je-krijgt-van-onze-helpdesk": "Bedankt voor je aanvraag. Onze helpdesk bekijkt deze binnen de vijf werkdagen. Heb je\n ondertussen nog vragen of toevoegingen met betrekking tot je aanvraag? Formuleer deze dan in\n een reply op automatische bevestigingsmail die je krijgt van onze helpdesk.",
```

Comment:
deze mogen niet vervangen worden
dit is wss een foutje in de translate extractor, ik onderzoek en doe een pr op jouw pr

Reply:
Klopt daar heb ik niets aan veranderd, ik heb gwn `npm run extract-translations` uitgevoerd